### ISSUE TITLE
problem about validation

### ISSUE THREAD

**Issue by 51265904017** [hms-dbmi/MOMA#4]
Thanks for your work. I have a question. If using model.eval() in validation, then the validation loss remains unchanged.  If not using model.eval() in validation, then the validation loss decreases a little but remains unchanged after several epochs. How can I solve the problem？
